app/javascript/selectors/answersSelector.js

```diff
@@ -1,10 +1,43 @@
 import {
   getAllActiveQuestions,
   getAllActiveQuestionsForCategory,
-  getAllQuestionsByCategoryId
+  getAllActiveQuestionIdsForCategory,
 } from "./questionsSelector";
 import { getCategoriesForBuilding } from "./categoriesSelector";
 
+export function getAnswersForBuilding(buildingId, state) {
```

Comment:
Let's rename this to `getUnfinishedAnswersForBuilding` instead so it's clear it doesn't return all answers! Same thing for any related methods